android/app/src/crypto/java/updated/mysterium/vpn/ui/top/up/crypto/payment/CryptoPaymentActivity.kt

```diff
@@ -118,14 +118,16 @@ class CryptoPaymentActivity : BaseActivity() {
     }
 
     private fun waitForPayment() {
+        val initialBalance = balanceViewModeL.balanceLiveData.value ?: 0.0
+        if (initialBalance >= 50.00) showPaymentBalanceLimitError(getString(R.string.payment_balance_limit_text, 50.00))
```

Comment:
Do we have AppConstants file? If not it would be good to have. 
At least we can add this value to companion object. Solution with AppConstants global object  better, but needs for time